features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -33,6 +33,8 @@
   "session.timeout.idle_session_timeout": "15m",
   "session.timeout.remember_me_session_timeout": "14d",
   "session.timeout.extend_remember_me_session_timeout_on_auth": true,
+  "par.cleanup.enable_expired_requests_cleanup": true,
+  "par.cleanup.clean_expired_requests_every": "1d",
```

Comment:
are these naming consistent with the other clean up configs?

Reply:
yes